README.md

```diff
@@ -102,6 +102,12 @@ If Che is behind Native Auth, local backend is prerequisite for Dev Server and t
 yarn frontend:start --env.server=http://localhost:8080/
 ```
 
+To avoid memory issues and the process being killed, more memory is possible through the following command in the frontend package directory:
+```sh
+$ NODE_OPTIONS="--max_old_space_size=6500" ./node_modules/.bin/webpack-dev-server --progress --color --config webpack.config.dev-server.js --env.server=http://localhost:8080/
+```
```

Comment:
what do you think about using 
```sh
$ NODE_OPTIONS="--max_old_space_size=6500" && yarn start --env.server=http://localhost:8080
 ``` 
 ?

Reply:
does it work ? I think at some point the environment variable  was not propagated ?
I'll retry if I reproduce the issue

Reply:
Locally I used next:
```sh
$ export NODE_OPTIONS=--max_old_space_size=4096
```
then
```sh
$ yarn start --env.server=http://localhost:8080
```
And it is worked for me.